protocol/rpc_status.go

```diff
@@ -98,7 +98,10 @@ func (rpc *RPCStatus) GetSuccessiveRequestFailureCount() int32 {
 
 // GetURLStatus get URL RPC status.
 func GetURLStatus(url common.URL) *RPCStatus {
-	rpcStatus, _ := serviceStatistic.LoadOrStore(url.Key(), &RPCStatus{})
+	rpcStatus, found := serviceStatistic.Load(url.Key())
```

Comment:
Why don't you use `LoadOrStore` directly? For avoid lock competition?

Reply:
It seems to have higher performance, go version is 1.13.
```go
func BenchmarkStore(b *testing.B) {
    for i := 0; i < b.N; i++ {
        _, found := methodStatistics.Load("test")
        if !found {
            _, _ = methodStatistics.LoadOrStore("test", &RPCStatus{})
        }
    }
}

func BenchmarkLoadOrStore(b *testing.B) {
    for i := 0; i < b.N; i++ {
        _, _ = methodStatistics.LoadOrStore("test", &RPCStatus{})
    }
}

BenchmarkStore-4                23008020                48.4 ns/op             0 B/op          0 allocs/op
BenchmarkLoadOrStore-4          10381435               102 ns/op              80 B/op          1 allocs/op
PASS
```

Reply:
yes, the new code will save the cost of "&RPCStatus{}", which only be used when the key is not set and do "store", and late access  to this key, it always "load". But each invoke will still cost for the "&RPCStatus{}".